readme.md

```diff
@@ -1,9 +1,9 @@
 # NestJS Elasticsearch Module
 
-## Introduction
-
 Welcome to Nestjs Elasticsearch module based on [@nestjs/elasticsearch](https://www.npmjs.com/package/@nestjs/elasticsearch) package.
 
+The current version (2.x) is fully compatible with Elasticsearch 8. For projects using Elasticsearch 7, use the previous version (1.x).

```

Comment:
Maybe 2.x should be replaces 2.x.x as we have also patch releases
And also Elasticsearch v7 and v8

Reply:
@adrianboratyn we have a support for v7 in v8? - am I understanding it right?

We can change 2.x to 2.x.x to indicate we have patch versions - but where comes the v7 and v8?

Reply:
Do we mean that the v7 and v8 support patch versions that are worth to mention in the readme?